packages/block-library/src/navigation/edit/use-more-navigation-menu.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import useNavigationMenu from '../use-navigation-menu';
+import useNavigationEntities from '../use-navigation-entities';
+
+const useMoreNavigationMenu = () => {
```

Comment:
I can't remember if I suggested this on the other PR. The naming of the hook seems to be tied to the UI rather than to the concept of what it represents which is "do we have either block or classic menus".

Also it's singular when the hook is looking for for multiple.

🙏 Please can we consider renaming to reflect the idea of the hook?
```suggestion
const useHasNavigationMenus = () => {
```